docs/adr/0008-refrain-from-using-docker-useremulation-in-nextflow.md

```diff
@@ -0,0 +1,23 @@
+# 8. Refrain from using docker userEmulation in Nextflow
+
+Date: 2022-06-02
```

Comment:
So far I've been marking these dates with `?` because I didn't know when in time they were made. Maybe you're right and it doesn't matter. Better to just give them _some_ date. I mean to replace the question mark but who knows when that will happen.

Reply:
As this was just discussed yesterday, I think its fair to use the date ;-)